frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -144,148 +140,140 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
       );
     };
     return (
-      <div className="modal-content modal-content--no-inner-scroll">
+      <form onSubmit={submit} name="form" className="modal-content modal-content--no-inner-scroll">
```

Comment:
Why do we need to use `form` in place of the Patternfly `Form` component?

Reply:
ack